servers/graphql-kotlin-ktor-server/src/test/kotlin/com/expediagroup/graphql/server/ktor/KtorGraphQLRequestParserTest.kt

```diff
@@ -89,13 +91,18 @@ class KtorGraphQLRequestParserTest {
     @Test
     fun `parseRequest should return request if method is POST`() = runTest {
         val mockRequest = GraphQLRequest("query MyFoo { foo }", "MyFoo", mapOf("a" to 1))
-        val serverRequest = mockk<TestApplicationRequest>(relaxed = true) {
-            every { call } returns mockk(relaxed = true) {
+        val serverRequest = TestApplicationRequest(
+            call = mockk<TestApplicationCall>(relaxed = true) {
+                every { application } returns mockk(relaxed = true) {
+                    every { receivePipeline } returns ApplicationReceivePipeline()
+                }
+                coEvery { receiveNullable<Any>(any()) } answers { callOriginal() }
```

Comment:
`receiveNullable()` (which is called from `ApplicationCall.receive()` in `KtorGraphQLRequestParser.parsePostRequest()`) is now a instance method instead of an extension function which is why `ApplicationCall.receive()` returns null when using a mock.
Using `callOriginal()` calls the instance method instead.

In Ktor 2.3.12
https://github.com/ktorio/ktor/blob/2.3.12/ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/request/ApplicationReceiveFunctions.kt#L95

In Ktor 3.0.x
https://github.com/ktorio/ktor/blob/main/ktor-server/ktor-server-core/common/src/io/ktor/server/application/PipelineCall.kt#L84